docs/dialect/ops_reference.rst

```diff
@@ -31,45 +32,1304 @@ The `CSR64` encoding is usually defined with the alias::
       indexBitWidth = 64
     }>
 
-The `CSC64` encoding can be defined with the alias::
+The `CSC64` encoding can be defined with the alias::
+
+    #CSR64 = #sparse_tensor.encoding<{
+      dimLevelType = [ "dense", "compressed" ],
+      dimOrdering = affine_map<(i,j) -> (j,i)>,
+      pointerBitWidth = 64,
+      indexBitWidth = 64
+    }>
```

Comment:
Typo: should be `#CSC64 = ...`

Reply:
https://github.com/metagraph-dev/mlir-graphblas/commit/7baac06ec1d63c98b347d8f8510655437321c9d9#diff-a32408ec551df7262d2f6412e9fba38b1c5e52207b4a2f51dfcbcdbf8ef17460R37 This change fixes this issue.